integration-tests/tests/specs/features/search/active-filters.spec.ts

```diff
@@ -41,7 +41,7 @@ test.describe('Search', () => {
         await searchPage.ebolaSudan();
 
         await page.getByPlaceholder('dd/MM/yyyy').first().click();
-        await page.getByTestId('calendar').getByText('11', { exact: true }).click();
+        await page.getByTestId('calendar').getByText('20', { exact: true }).click();
```

Comment:
Background (explained to me by @fhennig): The widget is now showing 11 April and 11 May, so the test doesn't know what exactly to click on:

<img width="261" alt="image" src="https://github.com/user-attachments/assets/eca14b42-056d-439e-b398-70437c61838e" />

This should not happen for the 20th.

Reply:
This will be submitted as a separate PR.